src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -16,24 +19,31 @@ import { getFormattedLocation } from 'src/shared/helpers/regionFormatter';
 import upperFirst from 'lodash/upperFirst';
 
 import styles from '../TrackPanelModal.scss';
+import { Bookmark } from '../../trackPanelState';
 
 type TrackPanelBookmarksProps = {
   activeGenomeId: string | null;
   genomeInfo: GenomeInfoData;
   exampleEnsObjects: EnsObject[];
+  bookmarks: Bookmark[];
+};
+
+type DispatchProps = {
   fetchExampleEnsObjects: (objectId: string) => void;
+  updateTrackStates: (trackStates: TrackStates) => void;
 };
+type Props = TrackPanelBookmarksProps & DispatchProps;
 
-export const TrackPanelBookmarks = (props: TrackPanelBookmarksProps) => {
-  const getExampleObjLabel = (exampleObject: EnsObject) => {
+export const TrackPanelBookmarks = (props: Props) => {
+  const getExampleObjLabel = (exampleObject: EnsObject | Bookmark) => {
     if (exampleObject.object_type === 'gene') {
       return exampleObject.label;
     } else {
       return getFormattedLocation(exampleObject.location);
     }
   };
 
-  const getPreviouslyViewed = () => {
+  const getExampleLinks = () => {
```

Comment:
This looks like a React component (`<ExampleLinks ensObjects={props.exampleEnsObjects} />`) rather than a function inside another component

Reply:
That's done